parcels/examples/tutorial_nemo_3D.ipynb

```diff
@@ -0,0 +1,138 @@
+      "INFO: Compiled JITParticleAdvectionRK4_3D ==> /var/folders/h0/01fvrmn11qb62yjw7v1kn62r0000gq/T/parcels-503/f8147345d9192dfa025490ff6045b52f.so\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "P[0](lon=3.349206, lat=51.770679, depth=1.011043, time=345600.000000)\n",
+      "P[1](lon=3.378664, lat=51.789280, depth=1.030178, time=345600.000000)\n",
+      "P[2](lon=3.403621, lat=51.805943, depth=1.081050, time=345600.000000)\n",
+      "P[3](lon=3.430356, lat=51.822876, depth=1.117308, time=345600.000000)\n",
+      "P[4](lon=3.456368, lat=51.839638, depth=1.150787, time=345600.000000)\n",
+      "P[5](lon=3.480758, lat=51.856472, depth=1.189697, time=345600.000000)\n",
+      "P[6](lon=3.507661, lat=51.873787, depth=1.213882, time=345600.000000)\n",
+      "P[7](lon=3.531766, lat=51.891018, depth=1.211578, time=345600.000000)\n",
+      "P[8](lon=3.559770, lat=51.908764, depth=1.192187, time=345600.000000)\n",
+      "P[9](lon=3.588396, lat=51.933567, depth=1.223015, time=345600.000000)\n"
+     ]
+    }
+   ],
+   "source": [
+    "from parcels import FieldSet, ParticleSet, JITParticle, AdvectionRK4_3D\n",
+    "from glob import glob\n",
+    "import numpy as np\n",
+    "from datetime import timedelta as delta\n",
+    "from os import path\n",
+    "\n",
+    "data_path = 'NemoNorthSeaORCA025-N006_data/'\n",
+    "ufiles = sorted(glob(data_path+'ORCA025-N06_200001??d05U.nc'))\n",
+    "vfiles = sorted(glob(data_path+'ORCA025-N06_200001??d05V.nc'))\n",
+    "wfiles = sorted(glob(data_path+'ORCA025-N06_200001??d05W.nc'))\n",
+    "mesh_mask = data_path + 'coordinates.nc'\n",
+    "\n",
+    "filenames = {'U': {'lon': mesh_mask, 'lat': mesh_mask, 'depth': wfiles[0], 'data': ufiles},\n",
+    "             'V': {'lon': mesh_mask, 'lat': mesh_mask, 'depth': wfiles[0], 'data': vfiles},\n",
+    "             'W': {'lon': mesh_mask, 'lat': mesh_mask, 'depth': wfiles[0], 'data': wfiles}}\n",
+    "\n",
+    "variables = {'U': 'uo',\n",
+    "             'V': 'vo',\n",
+    "             'W': 'wo'}\n",
+    "dimensions = {'U': {'lon': 'glamf', 'lat': 'gphif', 'depth': 'depthw', 'time': 'time_counter'},\n",
+    "              'V': {'lon': 'glamf', 'lat': 'gphif', 'depth': 'depthw', 'time': 'time_counter'},\n",
+    "              'W': {'lon': 'glamf', 'lat': 'gphif', 'depth': 'depthw', 'time': 'time_counter'}}\n",
+    "\n",
+    "fieldset = FieldSet.from_nemo(filenames, variables, dimensions)\n",
+    "\n",
+    "pset = ParticleSet.from_line(fieldset=fieldset, pclass=JITParticle,\n",
+    "                             size=10,\n",
+    "                             start=(2.96, 51.6),\n",
+    "                             finish=(3.26, 51.8),\n",
+    "                             depth=1)\n",
+    "\n",
+    "kernels = pset.Kernel(AdvectionRK4_3D)\n",
+    "pset.execute(kernels, runtime=delta(days=4), dt=delta(hours=6))\n",
+    "\n",
+    "print(pset)\n"
```

Comment:
What really is the point of this `print` statement? What do we want the users to take away? Why not a figure (that would also be useful for the card that we will create on http://oceanparcels.org/#tutorials)?